js/interface.js

```diff
@@ -801,14 +804,35 @@
     return customMenuLoadingPromise;
   }
 
+  function loadMenuManagerTab() {
+    var data = Fliplet.Widget.getData();
+
+    if (data && data.tab) {
+      $(`.nav.nav-tabs a[href='#${data.tab}']`).trigger('click');
+    }
+  }
+
   attachObservers();
   // Load menu widgets on startup
   loadCustomMenuWidgets();
+
+  loadMenuManagerTab();
+
+  Fliplet.Studio.onMessage((eventData) => {
```

Comment:
This is unnecessary because you're already listening to Studio events on line 92.

```suggestion
```